2-mandatory/2-financial-times.js

```diff
@@ -14,24 +20,53 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let shortestHeadline;
+    let fewestNumberOfWords;
+    for(let title of allArticleTitles){
+        let numberOfWords = title.split(' ').length;
+        if (fewestNumberOfWords === undefined || numberOfWords < fewestNumberOfWords){
+            fewestNumberOfWords=numberOfWords;
+            shortestHeadline = title;
+        }
+    }
+return shortestHeadline;
 }
 
+
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
```

Comment:
This looks good 👍 
I like that you used a separate function to check if the string contains a number - it makes the code more readable 😄 
One small point: when you have code like this - `if (containsNumbers(title)== true) {`, you can also write it like this `if (containsNumbers(title)) {`. Can you think of why that is?